build/entrypoint.sh

```diff
@@ -16,4 +16,8 @@ if [ ! -f "$BROKER_CONFIG" ] ; then
 fi
 echo "Using config file mounted to $BROKER_CONFIG"
 
-exec asbd -c $BROKER_CONFIG $FLAGS
+if [ ${DEBUG_ENABLED:-False} == "True" ]; then
+    dlv  --listen=0.0.0.0:${DEBUG_PORT} --headless=true --api-version=2 --log=true exec asbd -- -c $BROKER_CONFIG $FLAGS
```

Comment:
Could you skip modifying the entrypoint and simply add this to the debug Dockerfile?

Reply:
Adding this logic to the entrypoint production broker's use (including downstream) scares me.

Reply:
Yes can do I wasn't sure if this entrypoint.sh was used in production rpm